angular/src/lib/sidebar/sidebar.component.ts

```diff
@@ -56,8 +56,8 @@ export class SideBarComponent implements OnInit, OnDestroy {
   @Input() isPageLevel = false;
   /** @prop Sets padding for Topbar | false */
   @Input() withTopbar = false;
-  /** @prop Changes padding based on Icon nav | true */
-  @Input() withIcons = true;
+  /** @prop show minimized sidebar with icons only on toggle | true */
```

Comment:
Prop naming wise should this be more generic/explicit like shouldMinimize vs iconNav? @pauljeter